solutions/3529/01.py

```
@@ -0,0 +1,42 @@
+class Solution:
+    def countCells(self, grid: List[List[str]], pattern: str) -> int:
```

Ruff (0.14.10)

Undefined name `List`

(F821)